src/interfaces/libpq/fe-connect.c

```diff
@@ -4034,6 +4067,12 @@ freePGconn(PGconn *conn)
 	free(conn->inBuffer);
 	free(conn->outBuffer);
 	free(conn->rowBuf);
+	free(conn->oauth_issuer);
+	free(conn->oauth_discovery_uri);
+	free(conn->oauth_client_id);
```

Comment:
Is client_id and client_secret needed here? Since the token process is completely delegated to the client

Reply:
not required. Will remove this.